ui-admin/src/study/participants/DataChangeRecords.tsx

```diff
@@ -79,7 +81,9 @@ export default function DataChangeRecords({ enrollee, studyEnvContext }:
     {
       header: 'Source',
       cell: ({ row }) => (
-        row.original.responsibleUserId ? 'Participant' : 'Admin'
+        row.original.responsibleUserId ? 'Participant' :
+          row.original.responsibleAdminUserId &&
+            `Admin (${(users.find(u => u.id === row.original.responsibleAdminUserId)?.username)})`
```

Comment:
NICE! this is way easier and simpler than I thought it would be, you'll get a gold star from Parker for sure 😄